blocks/hero-block/README.md

```diff
@@ -0,0 +1,44 @@
+# @wpmedia/hero-block
+
+_Please provide a 1-2 sentence description of what the @wpmedia/hero-block is and what it does._
+
+## Props
+
+| **Prop**          | **Required** | **Type** | **Descripton** |
+| ----------------- | ------------ | -------- | -------------- |
+| **required prop** | yes          |          |                |
+| **optional prop** | no           |          |                |
+
+## ANS Schema
+
+Outline any schema information requirements necessary to know for ths block
+
+### ANS Fields
+
+- `Add all ANS fields used in the block`
+
+## Internationalization fields
+
+| Phrase key | Default (English)     |
+| ---------- | --------------------- |
+| `key`      | `english translation` |
+
+## Events
+
+Blocks can emit events. The following is a list of events that are emitted by this block.
+
+| **Event Name** | **Description**    |
+| -------------- | ------------------ |
+| **eventName**  | Describe the event |
+
+### Event Listening
+
+Include block specific intructions for event listening.
+
+OR
+
+This block does not emit any events.
+
+## Additional Considerations
+
+_This is optional. Please add an additional context that would be important to know in order to use this block._
```

Comment:
Are we requiring documentation here? @matthewroach @JackHowa

Reply:
This should be cleaned up and at minimum a small description on what the block is - props is not needed